akvo/templates/project_widgets2.html

```diff
@@ -29,28 +29,30 @@
                             <input type="radio" name="widget-project" value="project" checked>{% blocktrans with title=project.title %}Use the {{title}} project{% endblocktrans %}
                         </label>
                     </div>
-                    <div class="radio">
-                        <label>
-                            <input type="radio" name="widget-project" value="random">{% trans "Display a random project from an organisation" %}:
-                        </label>
-                    </div>
-                    <div class="form-group">
-                        <select class="form-control" id="widget-organisation" disabled>
-                        <option value="{{project.reporting_org.pk}}">{{project.reporting_org.name}}</option>
-                        {% for partner in project.all_partners.distinct %}
-                            {% if not partner == project.reporting_org %}
-                                <option value="{{partner.pk}}">{{partner.name}}</option>
-                            {% endif %}
-                        {% endfor %}
-                      </select>
-                    </div>
+                    {% if project.all_partners.distinct.count > 1 %}
```

Comment:
But this means if there's only one org associated with the project, the choice of a random project goes away.